src/Tools/Source/RunTests/Program.cs

```diff
@@ -220,7 +220,8 @@ private static IOrderedEnumerable<string> OrderAssemblyList(IEnumerable<string>
             obj["Is32Bit"] = !options.Test64;
             obj["AssemblyCount"] = options.Assemblies.Count;
             obj["CacheCount"] = result.CacheCount;
-            obj["ChunkCount"] = chunkCount;
+            obj["ChunkCount"] = partitionCount;
```

Comment:
NM on the duplication.  Thie code is correct.  I want to transition the name and need old + new until the code makes its way through the branches and webiset.